src/cli/generate-project/common/.gitignore

```diff
@@ -0,0 +1,5 @@
+/.idea
+/cache
+/node_modules
+/dist
+/build
```

Comment:
I think it can be more completed, some users can use VScode, etc

Reply:
I put only what I've seen, I think it can be good to not have a big gitignore and add it when needed, what do you feel about this suggestion ?

Reply:
I think about a VSCode part, something like for example: https://github.com/jhipster/jhipster-lite/blob/main/.gitignore#L79-L87